mitxgraders/formulagrader/matrixgrader.py

```diff
@@ -82,11 +98,29 @@ def schema_config(self):
             }): {
                 Required('is_raised', default=True): bool,
                 Required('msg_detail', default='type'): Any(None, 'type', 'shape')
-            }
+            },
+            Required('entry_partial_credit', default=0): Any(All(Number, Range(0, 1)), 'proportional'),
+            Required('entry_partial_msg', default=MatrixEntryComparer.default_msg): text_string
         })
 
+    @classmethod
```

Comment:
I don't see why this needs to be an independent method... I'm also a little wary that this is essentially saying that `MatrixGrader` doesn't benefit from the use of `set_default_comparer`, because its default comparer is always set through the config... Perhaps that's why you went with unvalidated here? If so, then it's weird for those config values to have defaults... I think we should probably talk through this one.

Reply:
`MatrixGrader` does benefit from `set_default_comparer` in the same way that `FormulaGrader` does. Note that `get_default_comparer` returns `cls.default_comparer` unless `entry_partial...` keys are used. 

The `entry_partial...` keys cause `self.default_grader` to be overridden, but only on a per-instance basis.

That said, I can see how the code looks misleading. I'll rename some things.

Reply:
Actually, I'm going to think a bit more about other ways to do this. This works, but it does require messing with the config about before validation occurs, which I don't love.